sql/dql/intro-joins/primary-foreign-keys-example.md

```diff
@@ -0,0 +1,68 @@
+---
+author: meridian
+
+type: normal
+
+category: must-know
+
+tags: 
+  - joins
+  - keys
+  - example
+
+practiceQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+
+
+---
+
+# Keys Example
+
+---
+
+## Content
+
+Consider the following two tables contained in the `pokedex` database.
+Here is the `pokemon_types` table:
```

Comment:
```suggestion
The `pokemon_types` table:
```

Since we say "Consider the following two tables..." I think we can say "The name_of_table table:" for both